packages/data/lib/di/injector_data.dart

```diff
@@ -1,5 +1,46 @@
+import 'package:data/configuration/base_options_configuration.dart';
+import 'package:data/interceptor/interceptor.dart';
+import 'package:data/repository/network_repository.dart';
+import 'package:data/services/api_base_service.dart';
+import 'package:data/services/service_payload.dart';
+import 'package:dio/dio.dart';
+import 'package:domain/repository/network_repository.dart';
 import 'package:get_it/get_it.dart';
 
 final getIt = GetIt.instance;
 
-void initInjectorData() {}
\ No newline at end of file
+void initInjectorData() {
+  _initModuleApi();
+  _initModuleRepository();
+}
+
+void _initModuleApi() {
+  GetIt.instance.registerSingleton<Dio>(
+    _buildMovieDio(),
+    instanceName: "movieDio",
+  );
+  GetIt.instance.registerSingleton<ApiBaseService<ServicePayLoad>>(
+    ApiBaseServiceImpl(GetIt.instance.get(instanceName: "movieDio")),
+    instanceName: "movieService",
```

Comment:
currently we have only one api service in the container, so it's not obvious to specify instance name